aten/src/ATen/test/quantized_test.cpp

```diff
@@ -0,0 +1,42 @@
+#include <gtest/gtest.h>
+
+#include <ATen/ATen.h>
+#include <iostream>
+#include <limits>
+#include <sstream>
+#include <cmath>
+#include <type_traits>
+#include <ATen/test/test_assert.h>
+// For quantize_uint8
+#include <ATen/quantized/Quantizer.h>
+
+using namespace at;
+
+TEST(TestQTensor, First) {
```

Comment:
nit before landing: If we add the second test in this file, should we name it as "TEST(TestQTensor, Second)"? Otherwise, maybe we can change "First" to some other names for this test (e.g., TestQuantize)?